CHANGELOG.md

```diff
@@ -5,9 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v0.3.1] 2019-06-14
+
+### Changed
+
+- Disabled ipvs collector.
+
+### Fixed
+
+- Fix monitored file system mount points.
+
+## [v0.3.0]
+
+### Changed
+
+- Upgrade to node-exporter 0.18.0.
+
 ## [v0.2.0]
 
 ### Added
 
 - Separate pod security policy for node-exporter and node-exporter-migration workloads.
 - Security context with non-root user (`nobody`) for running node-exporter inside container.
+
+[Unreleased]: https://github.com/giantswarm/kubernetes-node-exporter/compare/v0.3.1...HEAD
+[0.3.1]: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.3.1
+[0.3.0]: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.3.0
+[0.2.0]: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.2.0
```

Comment:
We should push some tags here.

Reply:
I think we haven't pushed tags is because this is using `appr` release channels. When we migrate to app CRs we'll definitely need to start pushing tags.

Should we start doing that now? @puja108 @pipo02mix WDYT?

Reply:
That's fine. Those would be only useful for URLs here.